src/app/store/pod/selectors.ts

```diff
@@ -223,13 +223,13 @@ const refreshing = createSelector(
 const groupByLxdServer = createSelector([lxd], (lxdPods) =>
   lxdPods.reduce<LxdServerGroup[]>((groups, lxdPod) => {
     const group = groups.find(
-      (group) => group.address === lxdPod.power_parameters.power_address
+      (group) => group.address === lxdPod.power_parameters?.power_address
     );
     if (group) {
       group.pods.push(lxdPod);
     } else {
       const newGroup = {
-        address: lxdPod.power_parameters.power_address,
+        address: lxdPod.power_parameters?.power_address,
```

Comment:
This throws an error on line 266; since `power_parameters` is possibly undefined, address can't then be used as an index type since this is also potentially undefined